lib/iris/experimental/um.py

```diff
@@ -687,6 +687,18 @@ def normalise(values, kind):
             sector_size = self._WORDS_PER_SECTOR * self._word_size
             for field in self.fields:
                 data = field.get_data()
+
+                if hasattr(field, "lbpack"):
+                    lbpack = field.lbpack
+                    packing = lbpack % 10
+                    compression = (lbpack // 10) % 10
+                    number_format = (lbpack // 1000) % 10
+                    if (packing not in (0, 1) or
```

Comment:
Packing can be 0 (unpacked) or 1 (WGDOS), I understand?

But otherwise, yes - that works too.

not (packing in (0,1) and compression == 0 and number_format in (0, 2, 3))